homeassistant/components/tesla/climate.py

```diff
@@ -13,7 +13,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-SUPPORT_HVAC = [HVAC_MODE_HEAT, HVAC_MODE_OFF]
+SUPPORT_HVAC = [HVAC_MODE_AUTO, HVAC_MODE_OFF]
```

Comment:
`AUTO` is meant for when the climate is on a schedule or being part of AI. I don't think that it's the case here? https://developers.home-assistant.io/docs/en/entity_climate.html#hvac-modes

Reply:
That's fair. `AUTO` seemed to match best when I reviewed it but `HEAT_COOL` is probably more in line with what other HVAC systems would follow. I will make that change and push it.

Reply:
For clarity, the UI of Tesla's climate system doesn't follow a lot of the standards that other HVAC systems follow. I was making the integration match the Tesla UI, not HA standards.

Reply:
By the standards laid out in the document you linked to the "off" state of Tesla's should actually be "HVAC_MODE_AUTO" because Tesla's automatically monitor the internal circuitry of the car turn the AC on when it gets too hot. They divert heating from the AC compressor to the batteries when they get too cold. My point in saying this is that Tesla are very strange and normal HVAC settings don't match up well.